addons/account_edi/models/account_edi_format.py

```diff
@@ -444,7 +444,7 @@ def _decode_attachment(self, attachment):
         is_text_plain_xml = 'text/plain' in attachment.mimetype and content.startswith(b'<?xml')
         if 'pdf' in attachment.mimetype:
             to_process.extend(self._decode_pdf(attachment.name, content))
-        elif 'xml' in attachment.mimetype or is_text_plain_xml:
+        elif re.search(r'xml$', attachment.mimetype) or is_text_plain_xml:
```

Comment:
We don't want `.mpkg`, `.svg`, `.xhtml` or `.xul` files. We also don't want Atom feeds `application/atom+xml`.
So checking for `application/xml` and `text/xml` is enough. Either with `.endswith('/xml')` (which I find more readable than a regex), either by checking `attachment.mimetype in ('text/xml', 'application/xml')` (or my least favorite: by adding `/` in the regex)

Reply:
Changes Done!